### PR TITLE
doc/rgw: refine "Setting a Zonegroup"

### PR DESCRIPTION
**Pull request by zdover23**
Streamline the "Setting a Zonegroup" section by separating out the necessary prerequisite reading from the procedure itself.

I also corrected a typo in the word "following" in an unrelated section.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/radosgw/multisite.rst [hunk 1]
```diff
@@ -1170,26 +1170,26 @@ a minimum, specifying the required settings:
...
+      radosgw-admin zonegroup set --infile zonegroup.json
 
-Where ``zonegroup.json`` is the JSON file you created.
+   Where ``zonegroup.json`` is the JSON file you created.
 
-.. important:: The ``default`` zonegroup ``is_master`` setting is ``true`` by
-   default. If you create a new zonegroup and want to make it the master
-   zonegroup, you must either set the ``default`` zonegroup ``is_master``
-   setting to ``false``, or delete the ``default`` zonegroup.
+   .. important:: The ``default`` zonegroup ``is_master`` setting is ``true`` by default. If you create a new zonegroup and want to make it the master zonegroup, you must either set the ``default`` zonegroup ``is_master`` setting to ``false``, or delete the ``default`` zonegroup.
```
Comment (anthonyeleven) [inline doc/radosgw/multisite.rst:1186]: s/new/additional

As written this is confusing if there are no existing instances.

**Review by anthonyeleven — APPROVED** [review]
See suggestion, approved to obviate subsequent handshake.

File: doc/radosgw/multisite.rst (hunk 1 above)
Comment (zdover23) [reply doc/radosgw/multisite.rst:1186]: You're not wrong. I'll fix it in this PR.

**Comment by zdover23** [comment]
The Reef branch already contains these fixes. No Reef backport is necessary.
https://github.com/ceph/ceph/pull/51072 - Quincy backport
https://github.com/ceph/ceph/pull/51073 - Pacific backport

**Pull request merged**